xmppserver/src/main/java/org/jivesoftware/openfire/spi/EncryptionArtifactFactory.java

```diff
@@ -367,6 +367,7 @@ public SslContext createServerModeSslContext(boolean directTLS) throws Unrecover
                 break;
         }
 
+        builder.trustManager(getTrustManagers()[0]); // The existing implementation never returns more than one trust manager.
```

Comment:
Should we also set the keyManager here?
`builder.keyManager(getKeyManagerFactory());`

Reply:
I don't think so, as the `keyManagerFactory` is already passed in de factory method of the `SslContextBuilder` isntance, a few lines higher.